GETTING_STARTED.md

```diff
@@ -44,6 +44,12 @@ Add HammerJS to your application via [npm](https://www.npmjs.com/package/hammerj
 (such as the [Google CDN](https://developers.google.com/speed/libraries/#hammerjs)), or served 
 directly from your app.
 
+For example, if you want to add it via npm:
+
+```bash
+npm install --save-dev @types/hammerjs
+```
```

Comment:
Sorry for the section, indeed I was talking about types.

As for @types/hammerjs, I had some missing hammer types errors on `ng serve` and I install this type because node_modules/@types/hammerjs was missing at that time :(
But now, with the fresh&latest nodejs and angular-cli, `ng new ...`, `npm install @angular/material`, `ng serve` and it's all good. So... we can close this. Thanks for your time.

angular-cli: 1.0.0-beta.16
node: 6.7.0
os: win32 x64